website/source/docs/enterprise/sentinel/index.html.md

```diff
@@ -9,9 +9,9 @@ description: |-
 # Nomad Enterprise Sentinel Policy Enforcement
 
 In [Nomad Enterprise](https://www.hashicorp.com/products/nomad/), operators can
-create [Sentinel policies](/guides/sentinel-policy.html) for fine grain policy
-enforcement.  Sentinel policies build on top of the ACL system and allow operators to define
-fine grain policies such as disallowing jobs to be submitted to production on
+create [Sentinel policies](/guides/sentinel-policy.html) for fine-grained policy
+enforcement. Sentinel policies build on top of the ACL system and allow operators to define 
```

Comment:
comma after system

Reply:
I believe the Oxford comma rules apply to a series of three or more items: https://en.wikipedia.org/wiki/Serial_comma

Reply:
Ah, thanks for clarifying.